gbpservice/neutron/services/servicechain/plugins/ncp/node_drivers/oneconvergence_service_node_driver.py

```diff
@@ -1062,6 +1072,10 @@ def _fetch_template_and_params(self, context, update=False):
                                   'param': stack_params})
         return (stack_template, stack_params)
 
+    def _get_service_profile_type(self, context):
+        #return context.current_node['description']  # description should contain ASAV or VYOS
+        return SERVICE_PROFILE_TYPES[0]     # For now return 'ASAV'
```

Comment:
We are still not sure, how to get service_profile. We discussed here, that for now we'll return ASAV as service profile, until we don't have a way to chose between vyos and asav.
Commented code using description field is only for reference.